R/plotutils.R

```diff
@@ -9,11 +9,11 @@ compute.plotlims <- function(val, logscale, cumul.dval, cumul.mdval){
   if(logscale) {
     tmp <- tmp[ tmp > 0 ]
     tmpp <- tmpp[ tmpp > 0 ]
-  }
-  if( ( all(is.na(tmp)) && all(is.na(tmpp)) ) | ( length(tmp) == 0 & length(tmpp) == 0 ) ){
-    warning("compute.plotlims: log scale requested but there are no positive data, setting default range\n")
-    tmp <- 10^(-6)
-    tmpp <- 10^2
+    if( ( all(is.na(tmp)) && all(is.na(tmpp)) ) | ( length(tmp) == 0 & length(tmpp) == 0 ) ){
```

Comment:
Why is this bracket clearly intended? The error message specifically states that logscale was requested, so this edge case handling is just fine in the `if`, isn't?

Reply:
Because the default range would be used if all data were negative even if no log scale were requested. This was intended purely in case of the log scale.